src/containers/map/MapUtils.js

```diff
@@ -1,12 +1,15 @@
 // @flow
 import { List, Map } from 'immutable';
+import { DataUtils } from 'lattice-utils';
 
 import { UNIT_CONVERSION } from './constants';
 
 import { PROPERTY_TYPES } from '../../utils/constants/DataModelConstants';
 
+const { getPropertyValue } = DataUtils;
+
 const getCoordinates = (entity :Map) :[number, number] => {
-  const [latStr, lonStr] = entity.getIn([PROPERTY_TYPES.LOCATION, 0], '').split(',');
+  const [latStr, lonStr] = getPropertyValue(entity, [PROPERTY_TYPES.LOCATION, 0], '').split(',');
```

Comment:
no change required here but what value is `getPropertyValue` providing over `getIn`? `getIn` already comes with a default value fallback. seems like a waste of instructions/cycles

Reply:
ok, but in what case would you use `getPropertyValue` then?

Reply:
Not sure. The syntax is exactly the same as calling getIn(...) but requires an extra import with a longer name. Seems like it goes through some effort to make sure you get a provided fallback value instead of `this` for empty paths that point to the root object. Doesn't seem like a problem that needs solving tho.

Reply:
getPropertyValue basically exists because multiple other projects had several versions of the same function already implemented, so I asked Giulia to add getPropertyValue to lattice-utils.

getPropertyValue is intended for entities. it uses get() and getIn() depending on the whether you pass an FQN or an array like [FQN, 0]. 

as for whether it provides much value over just using get() or getIn() directly... 🤷